pkg/database/migrations/31_scenes_captions.up.sql

```diff
@@ -3,5 +3,6 @@ CREATE TABLE `scene_captions` (
   `language_code` varchar(255) NOT NULL,
   `filename` varchar(255) NOT NULL,
   `caption_type` varchar(255) NOT NULL,
+  primary key (`scene_id`, `language_code`),
```

Comment:
I think this change broke our ability to support multiple captions formats for a scene.  
Ex:
scene.mp4
scene.en.vtt
scene.en.srt

Reply:
No worries. I've updated the migration to include the `caption_type` in the key.